app/controllers/api/v1/users.rb

```diff
@@ -0,0 +1,41 @@
+module API
+  module V1
+    class Users < Grape::API
+      include API::V1::Defaults
+      helpers do
+      end
+
+      resources :users do
+        desc "Return a user"
+        params do
+          requires :id, type: String
+        end
+        get ":id", root: "user" do
+          authenticate_user!
+          user = User.find params[:id]
```

Comment:
find hay find by bạn nhỏ

Reply:
@vanvtt-0952  chị ơi em bắt sự lỗi record not found ở default rồi, hay là bắt buộc phải xử lý ở đây ạ?